### PR TITLE
Update procedural-names.json

### PR DESCRIPTION
**Pull request by Oscar-Yam**
add melee weapon as PRIMARY_WEAPON

### CODE REVIEW

**Comment by kt0t0Sudd3n** [comment]
can you please make them be in a column?

**Comment by Oscar-Yam** [comment]
what do you mean by 'in a column'?

**Comment by ManzTellen** [comment]
only put one singular text per line. the same thing applies to the next line or so, as it will make them easier to read.

File: contribution/mobile/zh-TW/procedural-names.json [hunk 1]
```diff
@@ -257,38 +257,17 @@
 	"EQUIPMENT_TYPES_INSTRUCTIONS": "須要最少10個部件以啟用",
 	"EQUIPMENT_TYPES": {
 		"PRIMARY_WEAPON": {
-			"names": ["手槍", 
-				  "射線槍",
```
Comment (ManzTellen) [inline contribution/mobile/zh-TW/procedural-names.json:261]: Basically like this one (which are the line that you erased before)

File: contribution/mobile/zh-TW/procedural-names.json [hunk 2]
```diff
@@ -257,38 +257,17 @@
...
-				  "爆能手槍",
```
Comment (ManzTellen) [inline contribution/mobile/zh-TW/procedural-names.json:262]: Basically like this one (which are the line that you erased before)

Comment (Oscar-Yam) [reply contribution/mobile/zh-TW/procedural-names.json:262]: ok, I will do that next time. But how do edit the committed file?

**Comment by Oscar-Yam** [comment]
I managed to make it in  a column

**Pull request closed without merging**